### PR TITLE
Update DataTables to v 1.13.6 for flowcells page and add copy buttons extension

### PR DESCRIPTION
**Pull request by sofiahag**
Mattias requested a copy button for the flowcells page here: https://trello.com/c/sHymyqg3/1591-there-is-no-copy-table-option-for-flowcell-list-in-genstat

We might as well implement the new buttons while we're at it, right? :-) 

I only added the buttons for the Illumina FCs, not the ONT, do you think we need for ONT also?

### CODE REVIEW

**Comment by aanil** [comment]
> I only added the buttons for the Illumina FCs, not the ONT, do you think we need for ONT also?

Why not!

File: run_dir/design/flowcells.html [hunk 1]
```diff
@@ -254,7 +254,12 @@ <h1><span id="illumina_page_title">Illumina flowcells</span>
     var table = $('#fc_table').DataTable({
       "paging":false,
       "info":false,
-      "order": [[ 1, "desc" ]]
+      "order": [[ 1, "desc" ]],
```
Comment (aanil) [inline run_dir/design/flowcells.html:257]: Is this on stage?

Comment (sofiahag) [reply run_dir/design/flowcells.html:257]: It is now!

**Review by aanil — APPROVED** [review]
👍 Looks good!

**Pull request merged**